cpp/src/parquet/encryption/read_configurations_test.cc

```diff
@@ -22,6 +22,7 @@
 
 #include "arrow/io/file.h"
 #include "arrow/testing/gtest_compat.h"
+#include "arrow/util/config.h"
```

Comment:
Ever used?

Reply:
Yes this is needed so that `ARROW_WITH_SNAPPY` is defined, otherwise the tests below are always skipped. This is a bit unrelated to this change but I noticed this problem when running the tests locally.